### PR TITLE
ADCM-2357 expanded checks for UI config tests

### PR DESCRIPTION
**Pull request by annademina**

### CODE REVIEW

File: tests/ui_tests/app/configuration.py [hunk 1]
```diff
@@ -55,16 +68,40 @@ def assert_field_is_editable(self, field_element, editable=True) -> None:
             self.is_element_editable(field_element) == editable
         ), f"Field {field_element} should {'be editable' if editable else 'not be editable'}"
 
+    def get_api_value(self, adcm_credentials, app_fs, field: str):
+        """Gets field value by api"""
+
+        current_config = _make_request(
```
Comment (Sealwing) [inline tests/ui_tests/app/configuration.py:74]: Why do you want to get it directly via API?

File: tests/ui_tests/app/configuration.py [hunk 2]
```diff
@@ -55,16 +68,40 @@ def assert_field_is_editable(self, field_element, editable=True) -> None:
...
+            adcm_credentials, app_fs, "GET", f"/api/v1/{self.driver.current_url.replace(app_fs.adcm.url, '')}/current"
+        ).text
```
Comment (Sealwing) [inline tests/ui_tests/app/configuration.py:76]: why not `.json()`?

File: tests/ui_tests/app/configuration.py [hunk 3]
```diff
@@ -55,16 +68,40 @@ def assert_field_is_editable(self, field_element, editable=True) -> None:
             self.is_element_editable(field_element) == editable
         ), f"Field {field_element} should {'be editable' if editable else 'not be editable'}"
 
+    def get_api_value(self, adcm_credentials, app_fs, field: str):
```
Comment (Sealwing) [inline tests/ui_tests/app/configuration.py:71]: `get_config_field_value_by_api`, maybe?

File: tests/ui_tests/test_configs.py [hunk 4]
```diff
@@ -360,7 +360,11 @@ def test_configs_fields(sdk_client_fs: ADCMClient, config_dict, app_fs):
     fields = ui_config.get_app_fields()
     with allure.step('Check save button status'):
         save_err_mess = f"Correct status for save button {[expected['save']]}"
-        assert expected['save'] == ui_config.save_button_status(), save_err_mess
+        button_state = ui_config.save_button_status()
+        assert expected['save'] == button_state, save_err_mess
```
Comment (Sealwing) [inline tests/ui_tests/test_configs.py:364]: Assertion message

**Review by Sealwing — CHANGES_REQUESTED** [review]

File: tests/ui_tests/test_configs.py (hunk 4 above)
Comment (dgusakov) [reply tests/ui_tests/test_configs.py:364]: Take a closer look at `save_err_mess`

Comment (Sealwing) [reply tests/ui_tests/test_configs.py:364]: I've deleted this message :D Yeah, saw it right after commented

File: tests/ui_tests/app/configuration.py (hunk 1 above)
Comment (annademina) [reply tests/ui_tests/app/configuration.py:74]: how do you suggest?

File: tests/ui_tests/app/configuration.py (hunk 2 above)
Comment (annademina) [reply tests/ui_tests/app/configuration.py:76]: why json?

Comment (annademina) [reply tests/ui_tests/app/configuration.py:76]: why json?

Comment (Sealwing) [reply tests/ui_tests/app/configuration.py:76]: as I understand the next thing you do, you load json from text to present it as python dict, no?

File: tests/ui_tests/app/configuration.py (hunk 1 above)
Comment (Sealwing) [reply tests/ui_tests/app/configuration.py:74]: I think fields that you're using may be found via the client's `.confg(full=True)` call. I suggest checking it to reduce functionality duplication in code

Comment (annademina) [reply tests/ui_tests/app/configuration.py]: removed this func

File: tests/ui_tests/app/configuration.py (hunk 2 above)
Comment (annademina) [reply tests/ui_tests/app/configuration.py]: changed getting config

File: tests/ui_tests/app/configuration.py (hunk 3 above)
Comment (annademina) [reply tests/ui_tests/app/configuration.py]: changed name

**Review by Sealwing — APPROVED** [review]

**Review by dgusakov — APPROVED** [review]

**Review by Sealwing — APPROVED** [review]

**Review by AnatolyBobunov — APPROVED** [review]

**Review by alexterent — APPROVED** [review]

**Pull request merged**